python/MainModule.cpp

```diff
@@ -0,0 +1,43 @@
+
+/*
+ * Copyright (C) 2024 Intel Corporation
```

Comment:
Please use LLVM license style here. Reference: https://github.com/intel/graph-compiler/pull/85

Reply:
#85 is merged. Please rebase this PR and have a license check